test/box/gh-6405-discarding-arguments-in-box-func-call.test.lua

```diff
@@ -0,0 +1,45 @@
+args_type = function(...)                     \
+    local type_table = {}                     \
+    for i = 1, select('#', ...) do            \
+        type_table[i] = type(select(i, ...))  \
+    end                                       \
+    return type_table                         \
+end                                           \
+
+function TablesAreEqual(table1, table2)       \
+    for k, v in pairs(table1) do              \
+        if table2[k] ~= v then                \
+            return false                      \
+        end                                   \
+    end                                       \
+                                              \
+    for k, v in pairs(table2) do              \
+        if table1[k] ~= v then                \
+            return false                      \
+        end                                   \
+    end                                       \
+                                              \
+    return true                               \
+end                                           \
```

Comment:
Why don't you use `table.equals`? (https://github.com/tarantool/tarantool/commit/0afe1f78a51990d7849a11339fe9c5af06165bd4)

Reply:
Thanks for the comment. I was not aware of this feature. Fixed.